challenges/platform/troubleshooting.md

```diff
@@ -19,44 +17,46 @@ The interview team and candidate will join a scheduled zoom call for the trouble
+Our test is designed to cover a range of configuration errors and system faults encountered on a Linux server. Successful candidates will be familiar with collecting logs from and investigating unfamiliar software in order to diagnose underlying system problems. We avoid challenges that require application specific expertise; Instead, we use applications as a jumping off point for diagnosing underlying system issues.
 
 The faults are present in:
 - kernel / systemd / cgroups
-- filesystem or volumes
+- filesystems and volumes
 - networking
 - TLS connectivity
-- Application behaviour
+- Application behavior
 
 We use the following software to represent applications on the host, which are experiencing faults:
 - nginx
 - etcd
 - docker
+- kubernetes
 - ipvs
 
-As this is meant to test the way we normally work, you may use any resource you would normally use when troubleshooting a system; for example, using google as a reference is normal and expected.
+While it is not required to be familiar with these applications, familiarity will lend a leg up when you need to querying logs or judging expected behavior.
+
+This is meant to test the way we normally work; please use any resource you would use when troubleshooting a system at home or in your job. For example, it is encouraged to Google if you encounter a promising, but unfamiliar error message. We look favorably on candidates who know how to formulate good queries when they're stumped.
+
+Similarly, you're welcome to install troubleshooting packages on the servers. While we've installed many common debugging tools, if you're used to having `htop`, `tldr`, or `execsnoop` at your fingertips, don't hesitate to install them.
 
 ## Pitfalls and Gotchas
-Some of the challenges exercise some of the hardest problems we've seen in the wild, so it's common to get stuck on one problem. If you do find you're not making progress on an issue, it's fine to move on and skip a problem and come back to it later.
+Some challenges reproduce the hardest problems we've seen in the wild, so it's common to get stuck. If you don't know where to start or find you're not making progress on an issue, please skip that problem.  You may return to it later, after you complete the other problems.
 
-While this is a simulated environment, the candidate should treat the challenge as troubleshooting a live customer or production system.
+While this is a simulated environment, the candidate should treat the challenge as troubleshooting a live production or customer system. If you'd like to try something risky, ask the interview team and we'll act as your peers. We hold it against candidates that "cowboy" in a potentially dangerous guess at a fix without understanding the underlying problem.
```

Comment:
I'm worried how the particular phrasing of "we hold it against candidates" comes across. Maybe consider softening the language a little? Or even dropping the sentence altogether since I think the point might have already been made

Reply:
I also feel like this is getting into the territory of coaching the candidate on how to do better on the interview.

Reply:
4efe257 strikes the added sentence: `If you'd like to try something risky...`  I agree the wording was poor.

@knisbet 
> this is getting into the territory of coaching the candidate on how to do better on the interview.

I think there may be a minor philosophical point that this comment highlights.  I don't propose changing anything at the moment, but I do want to surface my thought process for discussion's sake. Particularly:

I believe in equipping candidates with all the knowledge they need to succeed.  Unwritten expectations feel like "gotchas" to me and I don't feel gotchas have a place in interviewing. In past debriefs we've made the call that cowboying fixes is greatly to the candidates detriment (to the point of passing on them or dropping the offer level), something that could cost them 25% TC or more. If there was a small thing I could do that would have such impact on an interview, I'd absolutely want to know about it.  Out of respect and empathy for candidates/future coworkers, I want them to know too.

The risk here is that candidates see strong wording and fake their performance, stopping to ask before they do risky things when they wouldn't have otherwise. To me, the risk of fakers feels less than the risk of a poor interview experience across candidates.  Tts quite hard to fake debugging skills in a high pressure environment like Minefield.  And if they can fake it that well, they may just fake it for the rest of their career here -- exemplifying learning. :laughing: A positive twist on "fake it 'till you make it"?

Personal anecdote: I was incredibly impressed with Facebook's PE interview prep (not the actual interview though, that was boring 5 part whiteboard BS). During the prep they told you exactly what you needed to know, to the point of suggesting reading about challenges on Glassdoor and exact problems on LeetCode.  I believe their theory was that expectations were high enough and interviewers are competent enough that any attempt to game the system would instead result in the candidate learning and becoming a better candidate/employee. Ever since going through FP's PE process, I try to hold my own  interview design and preparatory documentation to a similarly high bar.

/ramble

I suspect we're on the same page -- and it was just a poor initial wording that triggered the response.